planetaryimage/image.py

```diff
@@ -274,7 +285,10 @@ def _parse_data(self, stream):
             data_stream.close()
 
     def _parse_band_sequential_data(self, stream):
-        data = numpy.fromfile(stream, self.dtype, self.size)
+        if self.compression == 'none':
```

Comment:
if `compression` is `none` then use `numpy.fromfile` else `numpy.fromstring`.
Reason:
    numpy.fromfile returns '[]' for gzip or bz2 open file and won't throw any error.